tests/MongoDB.Driver.Examples/ChangeStreamExamples.cs

```diff
@@ -74,10 +75,10 @@ public void ChangeStreamExample2()
 
             // Start Changestream Example 2
             var options = new ChangeStreamOptions { FullDocument = ChangeStreamFullDocumentOption.UpdateLookup };
-            var enumerator = inventory.Watch(options).ToEnumerable().GetEnumerator();
-            enumerator.MoveNext();
-            var next = enumerator.Current;
-            enumerator.Dispose();
+            var cursor = inventory.Watch(options);
+            while (cursor.MoveNext() && cursor.Current.Count() == 0) { }
```

Comment:
Hm, do you think the comment from example 3 might be helpful here too?  😃 
`// keep calling MoveNext until we've read the first batch`

Reply:
Done